src/main/java/com/commercetools/sync/inventories/README.md

```diff
@@ -1,287 +1,122 @@
-function on a reference instance the `String` that represents supply channel key would be returned). By expanded reference
-we mean that calling [getObj()](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/Reference.java#L52)
-function on a reference's instance a [Channel](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/channels/Channel.jav)
-object that contains its key would be returned. More information about why it is important can be found in the
-[FAQ](#why-is-it-important-to-provide-extended-supply-channel-references-in-entries-from-input-lists-of-sync-process)
-section.
-
-#### `getStatistics`
-
-Used to get an object  containing all the stats of the sync process; which includes a report message, the total number
-of updated, created, failed, processed inventory entries and the processing time of the sync in different time units and in a
-human readable format.
-````java
-inventorySync.sync(inventoryDrafts);
-inventorySync.getStatistics().getCreated(); // 1000
-inventorySync.getStatistics().getFailed(); // 5
-inventorySync.getStatistics().getUpdated(); // 995
-inventorySync.getStatistics().getProcessed(); // 2000
-inventorySync.getStatistics().getReportMessage();
-/*
- * Summary of inventory synchronisation:
- * 2000 inventory entries were processed in total (1000 created, 995 updated and 5 failed to sync)."
- */
- ````
-
-### How to use `InventorySyncUtils` and `InventoryUpdateActionUtils`
-
-For users who want to perform sync processor by themselves the
-[InventorySyncUtils](https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/inventories/utils/InventorySyncUtils.java)
-and [InventoryUpdateActionUtils](https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/inventories/utils/InventoryUpdateActionUtils.java)
-utility classes are provided. Mentioned classes allow to match differences between `InventoryEntry` containing legacy
-data and `InventoryEntryDraft` containing current data and return update actions that should be executed on legacy
-entry to keep it up to date. To receive such
-[UpdateActions](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/commands/UpdateAction.java)
-you can use following methods:
-
-#### `buildChangeQuantityAction`
-Used for comparision of `quantityOnStock` values from provided entries. `Optional` that may contain updating
-[ChangeQuantity](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/commands/updateactions/ChangeQuantity.java)
-action is returned.
+**Preconditions:** The sync expects a list of `InventoryEntryDraft` objects that have their `sku` fields set.
```

Comment:
as suggested by @piobra [here](https://github.com/commercetools/commercetools-sync-java/commit/1ef9f6006f3bf00d14b6283b4cbd6cc3c2d155cd#commitcomment-22275650), I would add `otherwise the sync will trigger an errorCallback function set by the user (more on it can be found down below in the options` explanations)`.